plugins/decorators/client-references-links-decorator.js

```diff
@@ -0,0 +1,70 @@
+const X_PY_DOC_URLS_PROPERTY = "x-py-doc-url";
+const X_JS_DOC_URLS_PROPERTY = "x-js-doc-url";
+
+
+/**
+ * This decorator adds links to the Apify API Client libraries Python and JS references.
+ *
+ * The Apify API OpenAPI specfication has been enriched with Apify specifc vendor extensions 
+ * on `operation` and `tag` level to link the Apify Client functionality e.g. for `actorBuild_get`:
+ * ```
+ * x-js-parent: BuildClient
+ * x-js-name: get
+ * x-js-doc-url: https://docs.apify.com/api/client/js/reference/class/BuildClient#get
+ * x-py-parent: BuildClientAsync
+ * x-py-name: get
+ * x-py-doc-url: https://docs.apify.com/api/client/python/reference/class/BuildClientAsync#get
+ * ```
+ * 
+ * The prepended HTML example:
+ * ```
+ * <span style="float: right;">
+ *   <a href="https://docs.apify.com/api/client/python/reference/class/BuildClientAsync#abort" target="_blank" rel="noopener noreferrer">
+ *     Python doc
+ *   </a>
+ *   &nbsp;|&nbsp;
+ *   <a href="https://docs.apify.com/api/client/js/reference/class/BuildClient#abort" target="_blank" rel="noopener noreferrer">
+ *     JS doc
+ *   </a>
+ * </span>
+ *
+ * TODO: The HTML/CSS above will be subject of further design development, placeholder for now.
+ * ```
+ *
+ */
+function ClientReferencesLinksDecorator(target) {
```

Comment:
The function requires a little bit of lining.
I suggest some basic fixes, at least.